source/personal-growth/investment-time.html.md

```diff
@@ -3,7 +3,7 @@ title: Investment Time
 ---
 
 We dedicate 10% of our work time (~165 hours for full-time employees) every year to our personal
-growth and well-being—we call this Investment Time, and it's one of our longest-standing traditions.
+growth and well-being (not cumulative, resets at the end of the year). We call this Investment Time, and it's one of our longest-standing traditions.
```

Comment:
What about adding this info in the `(~165 hours for full-time employees)` part? It could be something like `(~165 not cumulative hours per year for full-time employees)`. Do you think it is clear enough?